ee/codegen/src/__test__/__snapshots__/workflow.test.ts.snap

```diff
@@ -195,6 +195,7 @@ exports[`Workflow > write > should generate correct code with multiple terminal
 from .inputs import Inputs
 from vellum.workflows.state import BaseState
 from .nodes.final_output import FinalOutput
+from .nodes.final_output_1 import FinalOutput
```

Comment:
Here you can see we generate unique module names, which is the goal of this PR.

We do still however generate duplicative class names without import aliases. This is a separate bug and will be fixed after this fern PR is merged and published: https://github.com/fern-api/fern/pull/5377

Reply:
I'll also add that this is an opinionated naming convention and I'm open to feedback/alternative ideas.

Basically the first time we encounter a name, we use it. Thereafter, we append with `_(idx+1)`.

Reply:
[super oos] do we anticipate allowing the user to specify class names for a node in the UI? could be an advanced feature way of addressing this issue long term.

Doesn't answer for the fact that even in that world, we need _some_ default behavior for if multiple nodes have the same class name specified anyway in its definition, which will be common early on